tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -39,6 +39,9 @@
   {% if ELASTICSEARCH_SCHEME == "https" %}"use_ssl": True,{% endif %}
   "host": "{{ ELASTICSEARCH_HOST }}",
   "port": {{ ELASTICSEARCH_PORT }},
+  {% if ELASTICSEARCH_USER != '' and ELASTICSEARCH_PASSWORD != '' %}
+  "http_auth": ("{{ ELASTICSEARCH_USER}}", "{{ ELASTICSEARCH_PASSWORD }}"),
```

Comment:
If we start supporting http auth in Elasticsearch, then we will have also to update all tutor plugins that connect to Elasticsearch (including forum). This is going to be non-trivial.